.circleci/config.yml

```diff
@@ -32,6 +32,8 @@ jobs:
             dbt seed --target postgres --full-refresh
             dbt run --target postgres --full-refresh
             dbt test --target postgres
+            dbt run --vars '{microsoft_auto_tagging_enabled: true}' --target postgres
```

Comment:
should this be a full refresh?

Reply:
(same Q for the other warehouse configs here)

Reply:
Yeah I shall update this.